code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MainActivity.kt

```diff
@@ -31,6 +32,11 @@ class MainActivity : AppCompatActivity() {
         btn_getLocalNotification.setOnClickListener {
             scheduleNotification(getNotification("Click on the notification for tracking"), 1000)
         }
+        btn_triggerIAM.setOnClickListener {
+            val contextData = HashMap<String, String>()
```

Comment:
Take advantage of Kotlin std library. You can create this map as `hashMapOf("testShowMessage": "true")`.